common/djangoapps/student/forms.py

```diff
@@ -3,6 +3,9 @@
 """
 from importlib import import_module
 import re
+import socket
```

Comment:
@lenacom Since edx.org and MITx are no longer using this view, but your instance is still using this view. I guess you can make the change in your own fork. I believe the plan is to retire this view after our next major version of edX is rolled out. Let me know if this opinion has no footing